app/src/main/java/com/github/gogetters/letsgo/MapsActivity.kt

```diff
@@ -10,9 +21,15 @@ import com.google.android.gms.maps.SupportMapFragment
 import com.google.android.gms.maps.model.LatLng
 import com.google.android.gms.maps.model.MarkerOptions
 
-class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
```

Comment:
Write UI tests using espresso in order to achieve more code coverage and ensure correctness